examples/dt/OSASINFRA/DFG-osasinfra-shiftstack_ci-multijob-ocp4.x.md

```diff
@@ -0,0 +1,73 @@
+| Node role                                     | bm/vm | amount |
+| --------------------------------------------- | ----- | ------ |
+| Openshift master/worker combo-node cluster    | vm    | 3      |
+| Compute nodes                                 | bm    | 3      |
+
+
+## Services, enabled features and configurations
+**Based on https://github.com/openshift/installer/tree/master/docs/user/openstack#openstack-requirements
+
+| Service                                     | configuration                                                | Lock-in coverage?  |
+| ------------------------------------------- | -------------------------------------------------------------| ------------------ |
+| HCI                                         |                                                              | Must have          |
+| Neutron                                     | ML2/OVN                                                      | Must have          |
+| Octavia                                     |                                                              | Must have          |
+| Swift                                       | rados-gw backed by Ceph(HCI)                                 | Must have          |
+| Keystone                                    | user/password and app creds                                  | Must have          |
+| Nova                                        | Metadata service enabled & NovaEnableRbdBackend disabled     | Must have          |
+| Glance                                      | RBD - backed by Ceph (HCI)                                   | Must have          |
+| Cinder                                      | RBD - backed by Ceph (HCI)                                   | Must have          |
+| Manila                                      | CephFS through NFS (ganesha)                                 | Must have          |
+| Barbican                                    |                                                              | Must have          |
+| ssl or tls-e e2e encrpytion                 |                                                              | Must have          |
+| ipv4 and ipv6 provider subnets              |                                                              | Must have          |
+
+## Considerations/Constraints
+
+1. OSASINFRA requires dictates specific lab
+2. Physical setups are required
+3. Specific switch configuration
+4. OSASINFRA testing matrix:
+    - Testing all the supported OCP releases for a given OSP release.
+    - Testing all the supported NetworkTypes per OCP releases for a given OSP release (OVNKubernetes and OpenShiftSDN).
+    - Testing supported installation types per OCP releases and NetworkType for a given OSP release (IPI, UPI, IPI-proxy, others)
+5. A subset of testing can be selected to be run depending on the above combinations.
+
+## Testing tree
+
+| Test framework                        | Stage to run |  Test case to report |
```

Comment:
reduce this to a subset of testing that validates OSP18. The whole job should take no more than 4 hours.